docs/building-custom-SKU.md

```diff
@@ -26,6 +26,7 @@ Build recipe must contain the following preprocessor definitions:
 | HAVE_MAT_ZLIB | on | Use zlib for HTTP requests compression. This option must always be turned on for any high-volume telemetry project |
 | HAVE_MAT_LOGGING | on | Enable internal SDK tracing / debug logging |
 | HAVE_MAT_WIN_LOG | off | Will log statements to disk on windows if trace enabled and HAVE_MAT_LOGGING defined |
+| HAVE_MAT_EVT_TRACEID  | off | Enable event tracking by adding trace-id to http request header on Windows |
```

Comment:
Can you bit add more context here  -  this is for debugging purpose, and not recommended to be enabled in production code. The collector doesn't parse/read this header. As of now, this is meant to be used through the capi, where the http-send handler is supposed to remove this header from the event data  before sending it to collector.

Reply:
Sure